<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QWidget" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWiDget {border-image: url(:/z/back3.jpg);}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>20</width>
     <height>511</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cut_Image">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background:transparent;
 color: rgb(0, 0, 0);
 border-style:outset;
 border:1px solid blue;
 border-radius:12px
}

QPushButton:hover{
 background-color: rgb(45, 104, 125);
 border-style: inset;
}

QPushButton:checked{ 
 background-color: rgb(6, 99, 130);
 color: blue;
}</string>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Device_Name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>左右X</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Ratio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Up_Down_Z">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>33</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_camera">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background:transparent;
 color: rgb(0, 0, 0);
 border-style:outset;
 border:1px solid blue;
 border-radius:12px
}

QPushButton:hover{
 background-color: rgb(45, 104, 125);
 border-style: inset;
}

QPushButton:checked{ 
 background-color: rgb(6, 99, 130);
 color: blue;
}</string>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_head_back_Y">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>上下Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>分辨率</string>
   </property>
  </widget>
  <widget class="QVideoWidget" name="Vedio_Area" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Left_Right_X">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut_iamge_num">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>前后Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageview">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border:1px solid blue</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>370</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_remove_Image">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>159</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
 background:transparent;
 color: rgb(0, 0, 0);
 border-style:outset;
 border:1px solid blue;
 border-radius:12px
}

QPushButton:hover{
 background-color: rgb(45, 104, 125);
 border-style: inset;
}

QPushButton:checked{ 
 background-color: rgb(6, 99, 130);
 color: blue;
}</string>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>相对位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>第</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alef</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 0, 255);</string>
   </property>
   <property name="text">
    <string>张</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
